### PR TITLE
RHDEVDOCS-875: Reviewed the Single-User: Deployment to Kubernetes sec…

### PR DESCRIPTION
**Pull request by supriya-bharadwaj**
…tion for language and asciidoc syntax.

Signed-off-by: Supriya Takkhi <sbharadw@redhat.com>

### What does this PR do?
Fixes lang. and markup issues in ** Che on Kubernetes -> Single-User: Deploy to Kubernetes**.
File name: `src/main/pages/setup-kubernetes/kubernetes-single-user.adoc`

Hi @rkratky , can you please review the PR? Thanks!

### CODE REVIEW

File: src/main/pages/setup-kubernetes/kubernetes-single-user.adoc [hunk 1]
```diff
@@ -8,24 +8,21 @@ folder: setup-kubernetes
 ---
 
 [id="supported-kuberentes-flavors-and-versions"]
-== Supported Kuberentes Flavors and Versions
+== Supported Kuberentes flavors and versions
```
Comment (kowen-rh) [inline src/main/pages/setup-kubernetes/kubernetes-single-user.adoc]: Typo: Kubernetes

File: src/main/pages/setup-kubernetes/kubernetes-single-user.adoc [hunk 2]
```diff
@@ -8,24 +8,21 @@ folder: setup-kubernetes
...
 
-Tested only on MiniKube v0.25 with vm providers VirtualBox and kvm2.
+The single-user deployment to Kubernetes is tested only on MiniKube v0.25 with the *VirtualBox* and *kvm2* VM providers.
```
Comment (kowen-rh) [inline src/main/pages/setup-kubernetes/kubernetes-single-user.adoc]: "MiniKube" should be "Minikube". This applies globally (I see three instances of "MiniKube" being used in the diff here).

Also, Minikube uses `docker-machine` driver plug-ins to provide functionality for hypervisors. I'd recommend changing "providers" to "drivers" or "driver plug-ins".

**Comment by supriya-bharadwaj** [comment]
Hi @rkratky , I have incorporated Robin's feedback in this PR. I think it is ready to be merged. Can you have a look, please? Thanks!

File: src/main/pages/setup-kubernetes/kubernetes-single-user.adoc [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Single-User&#58 Deploy to Kubernetes"
+title: "Single-User&#58 Deployment to Kubernetes"
```
Comment (rkratky) [inline src/main/pages/setup-kubernetes/kubernetes-single-user.adoc]: Should be consistent with multi-user: https://github.com/eclipse/che-docs/pull/536/files

**Review by rkratky — CHANGES_REQUESTED** [review]

**Comment by supriya-bharadwaj** [comment]
HI @rkratky , I have changed the name of the file to make it consistent with the multi-user one. I guess this one is now ready to be merged. Thanks!

**Comment by rkratky** [comment]
@supriya-bharadwaj, it's not about the file name but about the heading:  "Deploying multi-user Che to Kubernetes".

**Comment by supriya-bharadwaj** [comment]
@rkratky , yeah, I changed the heading to "Deploying single-user Che to Kubernetes". I should have added exactly that in my comment.

**Review by rkratky — APPROVED** [review]

**Pull request merged**